kafka-rest/src/main/java/io/confluent/kafkarest/response/StreamingResponseFactory.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.response;
+
+import com.fasterxml.jackson.databind.MappingIterator;
+
+public class StreamingResponseFactory {
```

Comment:
Optional: I would make this class an inner class of `StreamingResponse`.

Reply:
I thought Streaming response had enough inner classes, so made it separate :) No strong feelings though

Reply:
Ack